core/src/constants/address.rs

```diff
@@ -10,7 +12,7 @@ use derive_where::derive_where;
 
 /// This represents a constant address. Anything implementing this trait
 /// can be used to fetch constants.
-pub trait ConstantAddress {
+pub trait AddressT {
```

Comment:
I standardised on all addresses/payloads being either `AddressT` (just looking something up; constants & storage) or `PayloadT` (submitting/calling something; TX's & runtime APIs)

Reply:
perhaps modify the docs then above :point_up_2:

Reply:
What do you think needs modifying? I think they are still relevant? :)

Reply:
yepp, I misunderstood your comment